apps/web/app/(authenticated)/(app)/app/settings/user/update-user-email.tsx

```diff
@@ -258,7 +270,8 @@ export const UpdateUserEmail: React.FC = () => {
                 } catch (e) {
                   toast({
                     title: "Error",
-                    description: (e as Error).message,
+                    description:
+                      (e as ClerkError)?.errors[0]?.longMessage ?? "Error creating email address",
```

Comment:
indexing into the array would still fail if it has 0 length
```suggestion
                      (e as ClerkError)?.errors.at(0)?.longMessage ?? "Error creating email address",
```